doc-site/docs/tutorials/hello-world.md

```diff
@@ -0,0 +1,137 @@
+# Hello World with Paladin
+
+This tutorial walks you through deploying and interacting with a simple `HelloWorld` smart contract using the Paladin SDK. The example demonstrates how to:
+1. Deploy the contract,
+2. Interact with it by calling its `sayHello` function,
+3. Retrieve and verify the emitted event.
+
+---
+
+## Running the Example
+
+The example code can be found in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/example/public-storage).
+
+The storage solidity contract can be found [here](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/solidity/contracts/tutorial/Storage.sol).
+
+Follow the [Getting Started](../../getting-started/installation/) instructions to set up a Paladin environment. Then, follow the example [README](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/example/helloworld/README.md) to run the code.
+
+---
+
+### Overview
+
+We have a `HelloWorld` smart contract, which:
+- Emits a "welcome" message as an event when its `sayHello` function is called.
+
+### Key Artifacts
+To deploy and interact with the contract, we use:
+1. **ABI**: Describes the contract's interface, including its functions and events.
+2. **Bytecode**: The compiled contract code.
+
+These are pre-compiled and provided in the `helloWorldJson` object.
+
+---
+
+### Step 1: Deploy the Contract
+
+```typescript
+const deploymentTxID = await paladin.sendTransaction({
+  type: TransactionType.PUBLIC,       // Deploy publicly
+  abi: helloWorldJson.abi,            // ABI of the HelloWorld contract
+  bytecode: helloWorldJson.bytecode,  // Compiled bytecode
+  function: "",                       // No constructor arguments
```

Comment:
This isn't arguments - it would be the function name. Constructors in a smart contract are unnamed, so an empty string denotes to call the constructor.